tests/integration.js

```diff
@@ -9,6 +9,8 @@ require('../lib/impress');
 
 ncp.limit = 16;
 
+//let timer = setTimeout(taskExit, 5000);
```

Comment:
That's just to test CI, because it hangs on my laptop and I added timeout to exit, but it somehow works in Travis regardless of timeout. So, I'll investigate that before close PR.